intro-HPC/gent/macros.tex

```diff
@@ -3,20 +3,22 @@
 \macro{\university}{UGent}
 \macro{\association}{AUGent}
 % HPC definitions
-\macro{\hpc}{HPC-UGent}
+\macro{\hpc}{UGent-HPC}
```

Comment:
let's decide on this in https://github.com/hpcugent/vsc_user_docs/issues/87 and not try and fix it in this pull request.